core/src/database/organization/member/member.model.ts

```diff
@@ -40,6 +41,10 @@ export class Member extends BaseModel {
     @Field(() => [MemberRestriction])
     restrictions: MemberRestriction;
 
+    @Column()
```

Comment:
Why are these decorated?

Reply:
Had issues with this not being selected, might have to do with the typeorm 3 migration